Contribute/architecture-center/aac-contribute.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Contribute to the Azure Architecture Center
+description: This article describes how to contribute to the Azure Architecture Center.
+ms.topic: contributor-guide
+ms.prod: non-product-specific
+ms.custom: external-contributor-guide
+author: EdPrice-MSFT
+ms.author: edprice
+ms.date: 07/08/2022
+---
+
+# Azure Architecture Center contributions
+
+You can edit articles or create new articles for the Azure Architecture Center. The Azure Architecture Center (AAC) encourages contributions from the public, through GitHub issues and pull requests (PR). This page covers the general guidelines for contributing content to the Azure Architecture Center.
+
+## What is the Azure Architecture Center
```

Comment:
```suggestion
## What is the Azure Architecture Center?
```
I think this is ok as a question, but consider making this H2 simply "Overview".